packages/interchain/interchain-mock/src/interchain.rs

```diff
@@ -37,6 +37,7 @@ pub type MockBase<A> = cw_orch_mock::MockBase<A, MockState>;
 
 /// Interchain environment for cw_multi_test Mock environment
 /// This leverages Abstract's fork of cw_multi_test enabling IBC interactions
+#[derive(Clone)]
 pub struct MockInterchainEnvBase<A: Api> {
     /// Mock chains registered within the structure
     pub mocks: HashMap<String, MockBase<A>>,
```

Comment:
Gotta love `Rc<RefCell<_>>`, makes this clone so cheap and ensures we have the behavior we want.